dev-lang/kona/patches/kona-20190226.patch

```diff
@@ -0,0 +1,38 @@
+--- a/Makefile
++++ b/Makefile
+@@ -86,6 +86,14 @@ LDFLAGS += -lsocket
+ PRODFLAGS += -fast
+ endif
+ 
++ifeq (haiku,$(OS))
++CFLAGS += -lpthread
```

Comment:
Is this needed?

Reply:
I'm not sure; I added it because the most of the targets had `-pthread`.

Reply:
Haiku libpthread is empty, and is only there for compatibility purposes I believe. You can safely remove it.

Reply:
Ah, ok. I've removed it now.